File: gluecodium/src/test/resources/smoke/skip/output/android/com/example/smoke/ClassWithStructWithSkipLambdaInPlatform.java
/*

 *
 */

package com.example.smoke;

import com.example.NativeBase;

public final class ClassWithStructWithSkipLambdaInPlatform extends NativeBase {
    public static final class SkipLambdaInPlatform {
        public int intField;

        public SkipLambdaInPlatform(final int intField) {
            this.intField = intField;
        }


    }


    /**
     * For internal use only.
     * @hidden
     * @param nativeHandle The SDK nativeHandle instance.
     * @param dummy The SDK dummy instance.
     */
    protected ClassWithStructWithSkipLambdaInPlatform(final long nativeHandle, final Object dummy) {
        super(nativeHandle, new Disposer() {
            @Override
            public void disposeNative(long handle) {
                disposeNativeHandle(handle);
            }
        });
    }

    private static native void disposeNativeHandle(long nativeHandle);




}

File: gluecodium/src/test/resources/smoke/skip/output/android-kotlin/com/example/smoke/ClassWithStructWithSkipLambdaInPlatform.kt
/*

 *
 */

@file:JvmName("ClassWithStructWithSkipLambdaInPlatformExtensions")

package com.example.smoke

import com.example.NativeBase

class ClassWithStructWithSkipLambdaInPlatform : NativeBase {

    class SkipLambdaInPlatform {
        @JvmField var intField: Int



        constructor(intField: Int) {
            this.intField = intField
        }





    }



    /**
     * For internal use only.
     * @suppress
     * @param nativeHandle The handle to resources on C++ side.
     * @param tag Tag used by callers to avoid overload resolution problems.
     */
    protected constructor(nativeHandle: Long, @Suppress("UNUSED_PARAMETER") tag: Any?)
        : super(nativeHandle, { disposeNativeHandle(it) }) {}







    companion object {
        @JvmStatic private external fun disposeNativeHandle(nativeHandle: Long)
    }
}

